src/Form/Type/FormatterType.php

```diff
@@ -44,39 +41,18 @@ final class FormatterType extends AbstractType
     protected $translator;
 
     /**
-     * @var ConfigManagerInterface
+     * @var CKEditorConfigurationInterface
      */
-    protected $configManager;
-
-    /**
-     * @var PluginManagerInterface
-     */
-    protected $pluginManager;
-
-    /**
-     * @var TemplateManagerInterface
-     */
-    private $templateManager;
-
-    /**
-     * @var ToolbarManagerInterface
-     */
-    private $toolbarManager;
+    private $ckEditorConfiguration;
 
     public function __construct(
         PoolInterface $pool,
         TranslatorInterface $translator,
-        ConfigManagerInterface $configManager,
```

Comment:
Not sure if we should handle this as a BC break. The class is final, but you could create a new instance with the *old* signature

Reply:
@core23 
Don't really understand, this question is for me or the reviewers ?

Reply:
It's a question for the other @sonata-project/contributors

Reply:
To me it's a BC.

Everytime we had sometimes like this, we tried to support both way to instantiate the class, with a deprecation for the old way.

Reply:
any news on this @psyray? need some help to get into what's @VincentLanglet explained?

Reply:
@bmaziere Without news, you can try to finish the PR if you're interested ;)

Reply:
@VincentLanglet sure I can ;) I guess that as old classes/interfaces has been removed in 2.x of friendsofsymfony/ckeditor-bundle and if we want to keep the ability to instanciate the class the old way, we need to allow both version in composer.json right?

Reply:
Reconsidering the situation, I'm not sure we can call this a BC-break, since the we dropped support for CKEditor 1.0. To create new instance with the old signature, you'll need CKEditor 1.x isn't it ? So you'll not be able to install this version.
I'm not surprised by the fact that changes are expected when if the user update his CKEditor version.

WDYT @sonata-project/contributors ? Especially @core23 ?

Is it mandatory to keep supporting the v1 version and make the constructor compatible with both version ?

Reply:
IMHO it would be nice to upgrade first one bundle and then another. Otherwise to update to this version you need to update the FOSCKEditorBundle too. 

In this case the compatibility a maybe a bit hard, but it is doable (not like other updates we had to do previously)

Reply:
Ok, so @bmaziere, it would be nice to support `^1.0 || ^2.0`. 
And we would like to keep the ability to instanciate the class the old way and the new way.

You can see how we did (for another bundle) here: https://github.com/sonata-project/SonataAdminBundle/pull/6220/files

Reply:
@bmaziere
Thanks to finish my PR, It's holiday time for me :)